contracts/interfaces/vault-managers/parameters/IVaultManagerBorrowFeeParameters.sol

```diff
@@ -24,20 +24,27 @@ interface IVaultManagerBorrowFeeParameters {
      * @notice Sets the base percentage of borrow fee. Only manager is able to call this function
      * @param newBaseBorrowFee The borrow fee percentage (3 decimals)
      **/
-    function setBaseBorrowFee(uint32 newBaseBorrowFee) external;
+    function setBaseBorrowFeePercent(uint32 newBaseBorrowFee) external;
 
     /**
      * @notice Sets the percentage of the borrow fee for a particular collateral. Only manager is able to call this function
      * @param asset The address of the main collateral token
      * @param newEnabled Is custom fee enabled for asset
      * @param newFee The borrow fee percentage (3 decimals)
      **/
-    function setAssetBorrowFee(address asset, bool newEnabled, uint32 newFee) external;
+    function setAssetBorrowFeePercent(address asset, bool newEnabled, uint32 newFee) external;
 
     /**
      * @notice Returns borrow fee percentage for particular collateral
      * @param asset The address of the main collateral token
      * @return The borrow fee percentage (3 decimals)
      **/
-    function getBorrowFee(address asset) external view returns (uint32);
+    function getBorrowFeePercent(address asset) external view returns (uint32);
+
+    /**
+     * @notice Returns borrow fee for usdp amount for particular collateral
+     * @param asset The address of the main collateral token
+     * @return The borrow fee
+     **/
+    function calcBorrowFee(address asset, uint usdpAmount) external view returns (uint);
```

Comment:
The thing is the values here are not in per cents. They're in 1/1000ths of a per cent. So the names are misleading.
One may expect that if `getBorrowFeePercent(...)` returned 3 it means 3%.
I'd rather use basis points (0.01%) and name the functions like `getBorrowFee(address asset) external view returns (uint32 feeBasisPoints);` or `feeBP`.